data/test_analytics_run_env.yaml

```diff
@@ -0,0 +1,45 @@
+keys:
```

Comment:
OK, I know the table is a bit gnarly and also we're likely to recapitulate everything in an endpoint reference, so I'm future proofing by stuffing this info into YAML. You're welcome/I'm sorry.

Reply:
<3